src/client/types.ts

```diff
@@ -1,44 +1,29 @@
 import { McpServerConfig } from '../config/types.js';
 import { Client } from '@modelcontextprotocol/sdk/client/index.js';
 import { ToolSet } from '../ai/types.js';
-
-// Defining types as any because we can't import the types from the SDK
-export type VercelMCPClient = any;
-
-export type IMCPClient = Client | VercelMCPClient;
+import { GetPromptResult, ReadResourceResult } from '@modelcontextprotocol/sdk/types.js';
```

Comment:
nice! now that we have these types, let's also use them in `mcp-client.ts` instead of `any` for getPrompt and readResource